copy-this-template-directory/index.md

```diff
@@ -1,16 +1,1050 @@
 ---
-title: ''
-description: '120-160 characters'
+title: 'Auth with Directus and iOS'
+description: 'Directus auth for an iOS project'
 author:
-  name: ''
-  avatar_file_name: 'add-to-directory'
+  name: 'Harshpal Bhirth'
+  avatar_file_name: 'Harshpal.jpeg'
 ---
 
 ## Introduction
 
+In this tutorial, you will learn how to configure an iOS project that authenticates with Directus. You'll cover registering, logging in, logging out, viewing all posts from all users, creating a post, and editing and deleting posts from your user account.
+
 ## Before You Start
 
-<!-- ## Your Sections Here -->
+You will need:
+
+1. To have Xcode installed on your macOS machine.
+2. Knowledge of the Swift programming language.
+3. A Directus project - follow our [quickstart guide](docs.directus.io/getting-started/quickstart) if you don't already have one.
+
+## Setting Up Public Role
+
+1. Navigate to `Settings`.
+2. Select `Access Control`.
+3. Click on the `Public Role`.
+4. Access the `System Collection`.
+5. Locate `Directus_users`.
+6. Under the `Create` option, choose `Use Custom`.
+7. Enable field permissions for `Email` and `Password` options.
```

Comment:
```suggestion
4. Access the `System Collection`.
5. Locate `directus_users`.
6. Under the `Create` option, choose `Use Custom`.
7. Enable field permissions for `Email` and `Password` options.
```

Reply:
Add screenshot too, please, of the field permissions screen with just email and password selected.